treeherder/webapp/api/performance_data.py

```diff
@@ -110,3 +110,21 @@ def get_performance_data(self, request, project, jm, pk=None):
         data = jm.get_performance_series_from_signatures(signatures, interval_seconds)
 
         return Response(data)
+
+    @list_route()
+    @with_jobs
+    def platforms(self, request, project, jm, pk=None):
+        """
+        Get method implementation for platforms
```

Comment:
Let's say something more meaningful here, like "Return list of platforms for which we have performance data"